src/style.css

```
@@ -539,16 +485,20 @@ h2:hover a .anchor-hint {
 }
 
 .language-select {
-    padding-top: 0.15rem;
-    padding-right: 0.4rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 0.75rem;
```

Tiny suggestion to align the language switchers a bit nicer with the light/dark mode:

```suggestion
    gap: 0.75rem;
    margin-bottom: 8px;
```